Script/CI/check_coding_rule.py

```diff
@@ -687,40 +679,45 @@ def check_preprocessor_(path: str, code_lines: list) -> int:
     return flag
 
 
-# TODO: すべてのファイルがキャメルケースになったら追加する
 # 0: OK, 1: NG
-# def check_include_guard_(path: str, code_lines: list) -> int:
-#     basename, ext = os.path.splitext(os.path.basename(path))
-#     if not (ext == ".h" or ext == ".hpp"):
-#         return 0
-#     # print(path)
-#     # print(basename)
-#     # print(ext)
-#
-#     # 最初に発見したプリプロセッサディレクティブがインクルードガードであること，
-#     # さらにそれが適切な識別子であることを判断
-#
-#     for idx, line in enumerate(code_lines):
-#         if line.startswith("#"):
-#             # 初めて "#" を検出したものがインクルードガードでないとNG
-#             if idx + 1 >= len(code_lines):
-#                 return 1
-#
-#             line_ifndef = line
-#             line_define = code_lines[idx + 1]
-#             # print(line_ifndef)
-#             # print(line_define)
-#
-#             if not (line_ifndef.startswith("#ifndef ") and line_define.startswith("#define ")):
-#                 return 1
-#
-#             include_guard = basename.upper() + "_H_"
-#             return 0
-#
-#         else:
-#             continue
-#
-#     return 1
+def check_include_guard_(path: str, code_lines: list) -> int:
+    basename, ext = os.path.splitext(os.path.basename(path))
+    if not (ext == ".h" or ext == ".hpp"):
+        return 0
+
+    # 最初に発見したプリプロセッサディレクティブがインクルードガードであること，
+    # さらにそれが適切な識別子であることを判断
+
+    for idx, line in enumerate(code_lines):
+        if line.startswith("#"):
```

Comment:
このfor絶対プリプロセッサディレクティブ以外見ないだろうし早期リターン(continue)するようにした方がよさそう
```suggestion
        if not line.startswith("#"):
```

Reply:
たしかに

Reply:
そうしました